Rakefile

```diff
@@ -70,3 +70,28 @@ task :acceptance, [:project, :keyfile, :instance, :tests] do |t, args|
 
   Rake::Task["run"].invoke
 end
+
+desc 'Runs a simple ActiveRecord tutorial on a Spanner emulator.'
+task :example do |t|
+  t.libs << "examples/snippets"
```

Comment:
Got: `NoMethodError: undefined method `libs' for <Rake::Task example => []>:Rake::Task`

Reply:
Ah, that was a left-over from a previous attempt at configuring the samples, thanks for catching it. I've updated the task to actually execute a sample.